File: source/includes/usage-examples/connect-sample-app.kt
import com.mongodb.kotlin.client.MongoClient
import org.bson.Document

fun main() {

    // Start example code here

    // End example code here

    val database = mongoClient.getDatabase("admin")

    val command = Document("ping", 1)
    val commandResult = database.runCommand(command)
    println("Pinged your deployment. You successfully connected to MongoDB!")
}

File: source/includes/usage-examples/connect-sample-app.php
<?php

require __DIR__ . '/../vendor/autoload.php';

// Start example code here

// End example code here

try {
    $client->test->command(['ping' => 1]);
    echo 'Successfully pinged the MongoDB server.', PHP_EOL;
} catch (MongoDB\Driver\Exception\RuntimeException $e) {
    printf("Failed to ping the MongoDB server: %s\n", $e->getMessage());
}
